RELEASE_PROCESS.md

```diff
@@ -0,0 +1,113 @@
+<!--
+RELEASE_PROCESS.md
+Copyright 2016 Canonical Ltd.
+This work is licensed under the Creative Commons Attribution-Share Alike 3.0
+Unported License. To view a copy of this license, visit
+http://creativecommons.org/licenses/by-sa/3.0/ or send a letter to Creative
+Commons, 171 Second Street, Suite 300, San Francisco, California, 94105, USA.
+-->
+
+# Juju GUI Charm Release Process #
+
+## Updating the GUI ##
+
+If the new charm will have a new revision of the Juju GUI then before
+releasing the charm the GUI should have its version bumped and committed to
```

Comment:
I think the RELEASE_PROCESS could use a link to the semver docs so people know how to increment accordingly.